app/controllers/alchemy/admin/elements_controller.rb

```diff
@@ -78,7 +78,7 @@ def update
 
           Rails.logger.error "[ALCHEMY_LOCALE] updating contents for element in elements_controller#update: #{@element}"
           update_translations(@element) if @element.page.language.language_code == 'en'
-          update_skip_translate(params[:skip_translate])
+          update_skip_translate(params[:skip_translate]) unless params[:skip_translate].nil?
```

Comment:
I still want the method to execute if `params[:skip_translate] = false` so I used `.nil?`